chia/full_node/mempool_manager.py

```diff
@@ -605,11 +606,11 @@ async def new_peak(
             self.seen_bundle_hashes = {}
             for item in old_pool.spends.values():
                 _, result, err = await self.add_spend_bundle(
-                    item.spend_bundle, item.npc_result, item.spend_bundle_name, item.height_added_to_mempool
```

Comment:
I don't see this patch renaming `spend_bundle_name` in `MempoolItem`. Is this a synonym?
It seems unrelated to this PR